src/actions/vstudio/vs2010.lua

```diff
@@ -35,6 +35,28 @@
 
 
 
+--
```

Comment:
I'd prefer not to have two `generateSolution()` functions to juggle. Can this logic go in the 2005 version, with an `if _ACTION >= "vs2010"` if necessary?